### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by shahbazibnafzal**
1. Added product page
2. Added search box in the navbar for mobile view

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
Your experience in this field is reflected from the code and the UI Good work

**Review by sainath7878 — COMMENTED** [review]
Your code and UI looks good. Just that naming of css color variables could be better. color-dark, color-light does not give any meaning. Just a personal opinion though :)

**Comment by shahbazibnafzal** [comment]
> Your experience in this field is reflected from the code and the UI Good work

Thanks for the feedback @Dev-AkshatGupta :)

**Comment by shahbazibnafzal** [comment]
> Your code and UI looks good. Just that naming of css color variables could be better. color-dark, color-light does not give any meaning. Just a personal opinion though :)

Thanks for the feedback @sainath7878 , actually the color has been defined in the component library and I am using that in this project. Its not defined in this project. The reason I have used light and dark is because I'll be using light and dark mode and it will be convenient in that context.

File: css/index.css [hunk 1]
```diff
@@ -81,28 +71,12 @@ footer {
...
-.card-vertical-thin {
-    width: 30%;
-}
-
-.card-image-vertical {
-    width: 90%;
-    height: 20rem;
-}
-
 @media only screen and (max-width: 755px) {
```
Comment (mikesch803) [inline css/index.css:74]: this should be rem

**Review by mikesch803 — APPROVED** [review]
everything looks good

**Comment by shahbazibnafzal** [comment]
Thanks for the feedbacks and approval @Dev-AkshatGupta , @sainath7878 , @mikesch803 . Merging it to dev.

**Pull request merged**